### PR TITLE
Addon activation popup

### PR DESCRIPTION
**Pull request by chillburn**

### CODE REVIEW

File: src/components/popups/addon-activation.vue [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+            title,
+            okCallback
+          });
+      });
+    }
+  }
+};
+</script>
+<style lang="scss" scoped>
+.addon-activate-button {
```
Comment (auscaster) [inline src/components/popups/addon-activation.vue:160]: use .button-inline style on button instead

File: src/components/popups/addon-activation.vue [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+  max-width: 120px;
+}
+
+.addon-activation-popup {
+  --f7-block-header-font-size: 16px;
+}
+
+.block-strong {
```
Comment (auscaster) [inline src/components/popups/addon-activation.vue:168]: unused?

File: src/components/popups/addon-activation.vue [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+  },
+  methods: {
+    appName,
+    onClose() {
+      this.$emit('closed');
+    },
+    activateAddon() {
+      this.loading = true;
+      this.$store.dispatch('team/activateAddon', this.addonPackage).finally(() => {
+        const message = this.$t('addons.activate', {
```
Comment (auscaster) [inline src/components/popups/addon-activation.vue:136]: Lets clean this up a bit by inlining definitions, also they both use `addons.activate`?

```
        this.loading = false;
        const message = this.$t('addons.activate', {  addon: this.addonTitle });
        const text = this.$t('addons.activate', { addon: this.addonTitle });
```

**Comment by chillburn** [comment]
I've cleaned this up and removed un-used code. 
Also, there is a change in the tags getters.

File: src/components/inputs/select.vue [hunk 4]
```diff
@@ -14,7 +14,7 @@
       :info="hint"
       @input="updateValue($event.target.value)"
       :wrap="false"
-      :class="!this.modelValue ? {'select--placeholder': true} : null"
+      :class="!modelValue ? {'select--placeholder': true} : null"
```
Comment (auscaster) [inline src/components/inputs/select.vue:17]: `:class="modelValue && 'select--placeholder'`

File: src/components/popups/addon-activation.vue [hunk 5]
```diff
@@ -0,0 +1,171 @@
+<template>
```
Comment (auscaster) [inline src/components/popups/addon-activation.vue:1]: remove me

File: src/pages/activate.vue [hunk 6]
```diff
@@ -0,0 +1,120 @@
+<template>
```
Comment (auscaster) [inline src/pages/activate.vue:1]: rename to addon-activation

File: src/pages/activate.vue [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+}
+.activate-navbar {
+  --f7-navbar-inner-padding-left: 16px;
+  --f7-navbar-inner-padding-right: 16px;
+}
+.activate-grid-container .row {
+  max-width: 1024px;
+  margin: 0 auto;
+}
+.activate-navbar-preloader-button {
```
Comment (auscaster) [inline src/pages/activate.vue:112]: remove this

File: src/components/popups/addon-activation.vue (hunk 5 above)
Comment (chillburn) [reply src/components/popups/addon-activation.vue]: done

File: src/pages/activate.vue (hunk 6 above)
Comment (chillburn) [reply src/pages/activate.vue]: done. i've also renamed class names using .activate-xxxx to .addon-activation-xxxx

File: src/pages/activate.vue (hunk 7 above)
Comment (chillburn) [reply src/pages/activate.vue]: Done. Styles to fix button-fill in navbar have been done in /scss/navbar.scss

File: src/components/inputs/select.vue [hunk 8]
```diff
@@ -14,7 +14,7 @@
       :info="hint"
       @input="updateValue($event.target.value)"
       :wrap="false"
-      :class="!this.modelValue ? {'select--placeholder': true} : null"
+      :class="!modelValue && 'select--placeholder'"
```
Comment (auscaster) [inline src/components/inputs/select.vue:17]: `{'select--placeholder': !modelValue}`

File: src/pages/addon-activation.vue [hunk 9]
```diff
@@ -0,0 +1,112 @@
...
+    <f7-navbar class="addon-activation-navbar">
+      <f7-nav-title :title="$t(`${pkg}.title`)" />
+      <f7-nav-right>
+        <f7-button
+          fill
+          :text="$t('label.activate', 'Activate')" @click="activateAddon"
+          preloader
+          :tooltip="$root.tooltip(`${$t('label.activate', 'Activate')} ${$t(pkg + '.title')}`, false, loading)"
+          :loading="loading"
+          class="addon-activation-navbar-preloader-button"
```
Comment (auscaster) [inline src/pages/addon-activation.vue:12]: redundant

File: src/components/inputs/select.vue (hunk 8 above)
Comment (chillburn) [reply src/components/inputs/select.vue]: done

File: src/pages/addon-activation.vue (hunk 9 above)
Comment (chillburn) [reply src/pages/addon-activation.vue]: done.

**Pull request merged**